src/docs/010_manual/20_install.adoc

```diff
@@ -134,12 +148,11 @@ Run the following command to ask docToolchain to download the template of your c
 ----
 --
 
-== Build
+== Build: Creating Templates.
```

Comment:
I probably wouldn't use a prefix (Build:) in a section heading because it implies it's part of a series (Build: Creating Templates, Build: Something Else). No fullstops in headings either.

Also, the casing for headings is **T**itle **C**ase not **S**entence **c**ase. You will need to go back and change all your section headings to Title Case.

Reply:
Currently using [Microsoft's Sentence-style capitalization guide](https://docs.microsoft.com/en-us/style-guide/capitalization)